.github/workflows/publish.yaml

```diff
@@ -0,0 +1,56 @@
+name: Publish to pypi
+
+on:
+  push:
```

Comment:
I'm reading this as this action will run on all pushes.  It looks like the regular pypi upload may be limited to when we've just created a release tag.  

For testpypi, what will the version be?  I think there may be complications with this when we get rid of astropy helpers.